src/pages/integrations/shapeshift/shapeshift-shift/shapeshift-shift.ts

```diff
@@ -112,10 +109,9 @@ export class ShapeshiftShiftPage {
   }
 
   private showToWallets(): void {
-    this.toWallets =
-      this.fromWallet.coin == 'btc' ? this.walletsBch : this.walletsBtc;
```

Comment:
This is necessary because we can get an error like this: ```Pair btc_btc must contain two different input and output coins.```

toWallets and fromWallet can be the same, but in this case (shift) it must have different coin.

Reply:
reverted to master until we support ETH on shapeshift